Makefile

```diff
@@ -25,6 +25,7 @@ checks:
 	./scripts/checks "$(PKGS)"
 
 build: deps checks
+	rm -rf Godeps/_workspace/pkg
```

Comment:
Yeah, I understand. And I was hoping we could avoid clean before every build. 

Basically if we clean every time we are adding to the build time (as this will force a rebuild of vendored packages). 

And if I understand it correctly this cleanup is mostly needed when we are updating to a new version of vendored packages, right? So I was wondering if we move the `rm -rf ...` command to `clean` target then we leave it to the user to call `make clean` once every time they update godeps etc. Would that work?